src/Workspaces/CSharp/Portable/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -257,6 +257,14 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
                 return CreateAdjustSpacesOperation(0, AdjustSpacesOption.ForceSpacesIfOnSingleLine);
             }
 
+            // empty {} in pattern
+            if (previousToken.Kind() == SyntaxKind.OpenBraceToken &&
+                currentToken.Kind() == SyntaxKind.CloseBraceToken &&
+                !currentToken.Parent.IsKind(SyntaxKind.Block, SyntaxKind.ArrayInitializerExpression, SyntaxKind.NamespaceDeclaration, SyntaxKind.ClassDeclaration, SyntaxKind.StructDeclaration, SyntaxKind.InterfaceDeclaration))
```

Comment:
It would be simpler to just check for `SyntaxKind.PropertyPatternClause`, but that value is not available in `CodeStyle` and `Workspaces` projects, where I think this file is included.

Reply:
You can use `SyntaxKindEx.PropertyPatternClause`.